src/platforms/java/performance/instrumentation/open-feign.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: OpenFeign Integration
+sidebar_order: 20
+description: "Learn how to capture OpenFeign based HTTP clients performance"
+---
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+Sentry OpenFeign integration provides `SentryFeignClient` that creates a span for each outgoing HTTP request executed with a [Feign](https://github.com/OpenFeign/feign) based HTTP client.
+
+### Install
+
+```xml {tabTitle:Maven}
+<dependency>
+    <groupId>io.sentry</groupId>
+    <artifactId>sentry-openfeign</artifactId>
+    <version>{{ packages.version('sentry.java.openfeign', '5.1.0') }}</version>
+</dependency>
+```
+
+```groovy {tabTitle:Gradle}
+implementation 'io.sentry:sentry-openfeign:{{ packages.version('sentry.java.openfeign', '5.1.0') }}'
+```
+
+```scala {tabTitle: SBT}
+libraryDependencies += "io.sentry" % "sentry-openfeign" % "{{ packages.version('sentry.java.openfeign', '5.1.0') }}"
+```
+
+For other dependency managers see the [central Maven repository](https://search.maven.org/artifact/io.sentry/sentry-openfeign).
```

Comment:
```suggestion
For other dependency managers see the [central Maven repository](https://search.maven.org/artifact/io.sentry/).
```

when I open https://search.maven.org/artifact/io.sentry/ it does not work though, the search link probably has changed and we'd have to fix across the docs

eg https://search.maven.org/search?q=g:io.sentry

Reply:
https://search.maven.org/artifact/io.sentry/sentry-openfeign is a correct link. As I understand, this links to a page where you can look up syntax for adding dependency to other dependency managers, so a link to exact package makes sense..? That's how it is for other modules.

Reply:
for me it opens a blank page

Reply:
ahhh because its not published yet, my bad hehe